client/js/app/utils/ChartTypeUtils.js

```diff
@@ -0,0 +1,73 @@
+var _ = require('lodash');
+
+module.exports = {
+  getQueryDataType: function(query){
+    var isInterval = typeof query.interval === "string";
+    var isGroupBy = typeof query.group_by === "string" || (query.group_by instanceof Array && query.group_by.length === 1);
+    var is2xGroupBy = query.group_by instanceof Array;
+    var dataType;
```

Comment:
Moved each `var` declaration to own line, as specified in airbnb style guide: https://github.com/airbnb/javascript/tree/master/es5#variables